database/integration_test.go

```diff
@@ -494,7 +494,7 @@ func testHooks(t *testing.T, db Interface, resources *Resources) {
 	if err != nil {
 		t.Errorf("unable to list hooks for repo %d: %v", resources.Repos[0].GetID(), err)
 	}
-	if int(count) != len(resources.Hooks) {
+	if int(count) != len(resources.Hooks)-1 {
```

Comment:
do you mind adding a quick comment here on why it's `-1`? i know the reason, but it took me a minute :D